modules/dns/README.md

```diff
@@ -31,6 +31,49 @@ The returned `flags` parameter means that when you run a `terraform plan` comman
             # (12 unchanged attributes hidden)
         }
 
+The DNS records can be imported from a file in IBM Cloud Internet Services. This module allows the import of DNS records in two ways:
```

Comment:
See comments in https://github.com/terraform-ibm-modules/terraform-ibm-cis/pull/426

Reply:
Resolved all the comments.